bliss/models/decoder.py

```diff
@@ -195,6 +195,15 @@ def _render_ptiles(self, tile_catalog: TileCatalog) -> Tensor:
             tile_catalog["star_fluxes"], "b nth ntw s band -> (b nth ntw) s band"
         )
 
+        if "lens_params" in tile_catalog:
```

Comment:
we should talk about avoiding this `if` statements scattered throughout the code, in general tihs makes the code harder to read and maintain. I was imagining separate `lens_decoder` just like we have for the stars would be cleaner to maintain. Ofc it would have to be conditioned on the galaxy_bools or something. Could you tell me if this is possible ?